### ISSUE TITLE
Question: would this also make Interstate 82 playable?

### ISSUE THREAD

**Issue by CanadianJeff** [r1sc/Open76#26]
.

**Comment by Parapsycho** [comment]
TL; DR. It should be, if there is an interest in it. You just have to remember that this is project is about emulating the entire game engine from scratch, which is a monumental task for any game, let alone one made with proprietary software. Just be aware that Activision and Blizzard are, unfortunately, both hell-bent on holding on to their IPs, even when it doesn't make sense monetarily. 

I am personally bad at code, so I just play around with the art side of things when I can. I have just lost my files over many moves over the years. If I can remember all of the steps and have the time, I will post a video on how to extract models from the game using this project. I have other game-related projects in the works, so I '76 stuff has taken a back seat, unfortunately. I will also release any models I can find that I have extracted for free for the sake of transparency and furthering any development on the Interstate series. The process is long and arduous, but it is doable if you are meticulous enough. 

Interstate '76/Nitro/'82 used the same game engine Activision used to make Mech Warrior 3 and 4. So, you could theoretically use this project to make any game based off of the same game engine, but not games made on other engines. Activision actually developed Battle.Net before merging with Blizzard to work on Diablo II, which is why you see similarities between Interstate '82, Diablo II, and Auto Assault (RIP :( ) as far as inventory management is concerned. If you have the patience to mod Diablo II, that would be worth looking into also, but that is, unsurprisingly, a mess as well. Diablo II uses the same file structure, they just changed the file extensions to try and trick modders. 

The 'tricky' part to re-making any Interstate-related title is re-implementing the content creation tools (i.e Map making and skinning tools), as that was/is the central draw for most remaining players, all while maintaining the low-poly look that older gamers love. Everything else is just networking and physics. Battle.net has pretty much been figured out by the great minds over at Interstate76.com. This project is re-creating the game engine 

You can literally see where Activision re-used models from Interstate '76 Nitro in Interstate '82. The Ramps and tunnel models are direct rips from I '76 Nitro with different shaders applied to 'update' them to 1982.

Interstate '76/Nitro used proprietary software by Activision to make maps. 

Interstate '82 used an undeveloped version of Terragen to create it's maps. It was just scrapped early in development for a variety of reasons that are personal to each player. 

If you have any more questions, feel free to ask! 

Opinion time:
The Interstate Series is; for better or worse, a unique series of games that really hasn't gotten the attention it deserves over the years. It just didn't really adjust fast enough to the boom in gaming hardware/MMORPG's that occurred during the late 90's/Early 2000s, which is why you keep seeing Vigilante 8 re-makes pop-up from time to time, since that is the spiritual successor to the Interstate Series. But; that doesn't seem stop Activision or Blizzard from milking their other cash cows from time to time. I'm just a gamer at this point, so what do I know. lol

**Comment by CanadianJeff** [comment]
after that long winded answer not to mention knowledge about battle.net history you might be more than just a gamer, maybe even an ex developer that even worked on this game

I also kinda wanna fork this project and replace and i76 lines of code with i82 and see what the results are
